apps/hash-ai-worker-ts/src/workflows/run-flow-workflow.ts

```diff
@@ -112,7 +112,7 @@ export const runFlowWorkflow = async (
   const flow = initializeFlow({
     flowDefinition,
     flowTrigger,
-    flowId: workflowId as EntityUuid,
+    flowId: workflowId as EntityId,
```

Comment:
Is this an `EntityId`? In the `startFlow` resolver we're setting this via `generateUuid()` (i.e. without the `ownedById`).

Also this is something we can address in follow-up, but just want to flag the types here are currently incorrect.

Reply:
I've reverted this